File: 250721/남녀노소 구분짓기/sex-and-age.py
# 입력
intSex = int(input())
intAge = int(input())

if intSex == 0 : # 남자
    if intAge >= 19 : # 성인
        print("MAN")
    else :
        print("BOY")    
else :
    if intAge >= 19 :
        print("WOMAN")
    else :
        print("GIRL")

File: 250721/남녀노소 구분짓기/sex-and-age.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intSex = sc.nextInt();
        sc.nextLine();
        int intAge = sc.nextInt();

        // 변수 정의
        String strResult;

        // 조건문
        if (intSex == 0) {
            if (intAge >= 19) {
                strResult = "MAN";
            } else {
                strResult = "BOY";
            }
        } else {
            if (intAge >= 19) {
                strResult = "WOMAN";
            } else {
                strResult = "GIRL";
            }
        }

        // 출력
        System.out.println(strResult);
    }
}